bin/deploy_assets

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+set -e
+
+git checkout master
+git pull
+bin/version_check
+gulp deploy_asset
```

Comment:
Is `gulp deploy_asset` supposed to be `gulp deploy_assets` (with the s on the end)